### PR TITLE
1335: edits to new commands.rst file, adding cross-refs

### LINKED ISSUE #1335: DOC: edits and cross-refs for new commands.rst file
Edits and cross-refs for new commands.rst file--smooth out the intro text for the sections (it's mostly a list of commands) and figure out the best places to link to this file from various doc locations.

For background, see Slack chat, sonar pilot channel, May 2.

Also, see issue #1310 for more information.

### PR DESCRIPTION
**Pull request by jlprevatt**
Please review the changes. Thanks.

Closes #1335

### CODE REVIEW

File: docs/source/quick_start.rst [hunk 1]
```diff
@@ -31,6 +31,8 @@ Creating Credentials and Sources for a Network
 
 A *network* source is composed of one or more host names, one or more IP addresses, IP ranges, or a combination of these network resources. In addition, the source creation command references one or more credentials. Typically, a network source might include multiple credentials because it is expected that many credentials would be needed to access a broad IP range.
 
+**IMPORTANT:** For a network scan, Quipucords must use the credentials to run some commands with sudo access. For more information about this requirement, see `Commands Used in Scans of Remote Network Assets <commands.html>`_
+
```
Comment (kdelee) [inline docs/source/quick_start.rst]: Nitpick: the ever present threat that someone is using one of the other "become" methods that is not `sudo`
`... to run some commands that require escalated privileges. This is typically provided by use of sudo. For more information ...`

File: docs/source/working_with_sources.rst [hunk 2]
```diff
@@ -15,6 +15,8 @@ The following scenarios provide examples of how you would create a network, vcen
 .. _network:
 Creating a Network Source
 ^^^^^^^^^^^^^^^^^^^^^^^^^
+**IMPORTANT:** For a network scan, Quipucords must use the credentials to run some commands with sudo access. For more information about this requirement, see `Commands Used in Scans of Remote Network Assets <commands.html>`_
+
```
Comment (kdelee) [inline docs/source/working_with_sources.rst]: same comment as above

File: docs/source/requirements.rst [hunk 3]
```diff
@@ -5,5 +5,5 @@ Before installing Quipucords in your environment, review the following guideline
...
 - Any network sources that are targeted for the inspection process must be running SSH.
-- The user account that Quipucords uses for the SSH connection into the target systems, otherwise known as the credential, must have adequate permissions to run commands and read certain files, such as privilege escalation required for the ``systemctl`` command.
-- The credential user account requires an ``sh`` shell or a similar shell. For example, the shell *cannot* be the ``/sbin/nologin`` or ``/bin/false shell``.
+- For a scan of network systems, the user account (credential) that Quipucords uses for the SSH connection into the target systems must have adequate permissions to run commands and read certain files on those systems. For example, some of the commands require privilege escalation to gather the complete set of facts for the scan. For more information about these commands, see `Commands Used in Scans of Remote Network Assets <commands.html>`_
+- The credential user account requires an ``sh`` shell or a similar shell. For example, the shell *cannot* be the ``/sbin/nologin`` or ``/bin/false`` shell.
```
Comment (kdelee) [inline docs/source/requirements.rst]: This sounds good!

File: docs/source/commands.rst [hunk 4]
```diff
@@ -1,10 +1,16 @@
-Remote Scan Commands
-~~~~~~~~~~~~~~~~~~~~~
-This section documents required commands on the remote host for network scans.  In addition to the commands below, we depend on standard shell facilities like those provided by bash.
+Commands Used in Scans of Remote Network Assets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When you run a network scan, Quipucords must use the credentials that you provide to run certain commands on the remote systems in your network. Some of those commands must run with the sudo command. That escalated privilege is required to gather the types of facts that Quipucords uses to build the report about your installed products and consumed entitlements.
+
+Although it is possible to run a scan for a network source without sudo access, the results of that scan will be incomplete. The incomplete results from the network scan will affect the accuracy of the generated report for the scan.
```
[the remark this answers is not in the document]
Comment (jlprevatt) [reply docs/source/commands.rst]: In the construction of the sentence, the accuracy refers to the report, not the facts. And a report based on incomplete facts will be an inaccurate report (whether you think about it in terms of "correctness"--1st meaning, or "exactness"--2nd meaning, Webster's and American Heritage). 

However, I changed the "accuracy" to "quality." I don't think that word is quite as good, but I resolved to change it if I could find an okay substitute quickly.

File: docs/source/requirements.rst (hunk 3 above)
Comment (jlprevatt) [reply docs/source/requirements.rst]: thanks

File: docs/source/quick_start.rst (hunk 1 above)
Comment (jlprevatt) [reply docs/source/quick_start.rst]: "sudo access" changed to "elevated privileges"

File: docs/source/working_with_sources.rst (hunk 2 above)
Comment (jlprevatt) [reply docs/source/working_with_sources.rst]: "sudo access" changed to "elevated privileges"

File: docs/source/requirements.rst (hunk 3 above)
Comment (jlprevatt) [reply docs/source/requirements.rst]: Based on Slack conversation about whether "escalation" had a negative connotation RE cyber attacks, changed it to "elevation"

**Pull request merged**